pkg/go.mod

```diff
@@ -162,3 +162,5 @@ require (
 	gopkg.in/warnings.v0 v0.1.2 // indirect
 	gopkg.in/yaml.v2 v2.2.8 // indirect
 )
+
+require github.com/rogpeppe/go-internal v1.8.1 // indirect
```

Comment:
Should this be moved into the main require block?

Reply:
This change is made by `go mod tidy` command so apparently this is how it likes it, I suggest we keep it to reduce diffs from when repeatedly applying `go mod tidy`.